test/pkg/environment/common/expectations.go

```diff
@@ -503,41 +504,28 @@ func (env *Environment) ConsistentlyExpectNodeCount(comparator string, count int
 	return lo.ToSlicePtr(nodeList.Items)
 }
 
-// ConsistentlyExpectNoDisruptions ensures that the state of the cluster is not changed within a passed duration
-// Specifically, we check if the cluster size in terms of nodes is the same as the passed-in size and we validate
-// that no disrupting taints are added throughout the window
-func (env *Environment) ConsistentlyExpectNoDisruptions(nodeCount int, duration time.Duration) {
+func (env *Environment) ConsistentlyExpectDisruptingNodesWithNodeCount(nodesTainted int, nodeCount int, duration time.Duration) []*v1.Node {
```

Comment:
We didn't need it because we didn't have any replacement tests. I was thinking of adding in a `replacementActions int` into the parameters that would assert that the node count at each point is at least `startingNodeCount` and at most `startingNodeCount + numReplacements`, thoughts?

Reply:
Yeah, I think I'd prefer that interface. I'd also personally prefer that we keep around the other function that asserts that we don't have any disruptions, even if it just calls out to your function that you've just written here under the hood.

From an interface perspective, I think it's a little awkward to say that I'm expecting a certain number of disruptions, but that number is actually 0 and I'm really not expecting any disruptions.